gitdash/pages/api/github.ts

```diff
@@ -0,0 +1,58 @@
+const { Octokit } = require('@octokit/rest')
+
+export default async (_req: any, res: { status: (arg0: number) => { (): any; new(): any; json: { (arg0: { stars: any; followers: any; starred: any; all_repos: any; repo_count: any; issues: any; issue_count: any }): any; new(): any } } }) => {
+  const octokit = new Octokit({
+    // auth: 
+    // github token for a particular user, leaving empty for now
+  })
+  const username = "prakharrathi25"    // need to get the input from the user for this  
```

Comment:
This is fine for an MVP; I suggest you work on making this dynamic this week though!

Reply:
Yup, that was the plan and we have achieved that this week. 😄